bench/testserver_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	"github.com/lytics/grid/v3"
 	"github.com/lytics/grid/v3/testetcd"
-	"go.etcd.io/etcd/clientv3"
+	clientv3 "go.etcd.io/etcd/client/v3"
```

Comment:
[Q] How would this be referenced if we didn't do the alias? Would it be `client`? That might be cleaner.